src/TU-Berlin-2.0.xml

```diff
@@ -6,8 +6,7 @@
     </crossRefs>
     <text>
       <copyrightText>
-        <p>Copyright
-		      <alt match=".+" name="copyrightYears">1992, 1993, 1994</alt> by Jutta Degener and Carsten Bormann,
+        <p>Copyright 1992, 1993, 1994 by Jutta Degener and Carsten Bormann,
 	      </p>
```

Comment:
does this need a new <p> for the last name?

Reply:
Not sure I understand - there is a paragraph separating the last name and the university